neofetch

```diff
@@ -3978,27 +3984,31 @@ prin() {
     if [[ "$(trim "$1")" && "$2" ]]; then
         [[ "$json" ]] && { printf '    %s\n' "\"${1}\": \"${2}\","; return; }
 
-        string="${1}${2:+: $2}"
+        string="${1}"
```

Comment:
Why was this removed?

Reply:
The way things are printed in this PR makes concatenating arg2 to arg1 redundant, but arg1 is still formatted correctly.

Reply:
I have a more complete explanation now. 

Originally, this if statement joined the two arguments into the same string, but to properly align the info by label, the labels and their corresponding data had to stay separate. I've left the JSON logic in place, but set string to the first argument in both cases, so I've updated the code to be 
```
# If $2 doesn't exist we format $1 as info.
if [[ "$(trim "$1")" && "$2" ]]; then
    [[ "$json" ]] && { printf '    %s\n' "\"${1}\": \"${2}\","; return; }
fi
string="${1}"
string="$(trim "${string//$'\e[0m'}")"
```

Now I'm not really sure what the lines 
```
length="$(strip_sequences "$string")"
length="${#length}"
```
do, but removing them doesn't seem to change anything, so I've left them in. 

Anyway, not concatenating $2 to $1 has several benefits:

First, we still strip all sequences from the labels, standardize them to bold, and get the length.

Second, now I can pad the first argument of calls to prin with an arbitrary number of `\033[0m  \033[0m  ...` and manually center text if I want to give my own short byline like this screenshot: 
<img width="479" alt="Screenshot 2019-11-28 at 00 44 28" src="https://user-images.githubusercontent.com/33698018/69783030-420b3580-1178-11ea-9f4f-3d7735583fb8.png">

Third, we can still properly detect automatically whether or not the user probably wants a separator at the end of the string with 
```
string="${string}${reset}${colon_color}"
[[ "$2" ]] && string="${string}${separator}${separator:=:}"
```

Fourth: Now users can use arbitrary sequences to format text the way they like (italic, flashing, etc). 

With the string formatting simplified we can do:

```
printf '%b' "${text_padding:+\e[${text_padding}C}${zws}"
case "$align_output" in
    "left")  printf '%-*b' $offset "${string}" ;;
    "right") printf '%+*b' $offset "${string}" ;;
    "off") printf '%b' "${string}" ;;
esac
[[ "$2" ]] && printf '%b' "${info_color}${2}"
printf '\n'
```

If any of this breaks existing behavior I haven't witnessed it, so I'll need feedback from people with more exotic setups.